rustplus/api/remote/rust_remote_interface.py

```diff
@@ -132,7 +132,9 @@ async def get_response(
 
                 else:
 
+                    await self.api.lock.acquire()
                     await self.send_message(app_request)
+                    self.api.lock.release()
```

Comment:
```suggestion
                    async with self.api.lock:
                        await self.send_message(app_request)
```

Reply:
Could simplify some code if you use the context manager

Reply:
Good Point, often forget about the context managers. Will change this before merging